CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - SALE ORDER LINE: Fix NPE in groovy script.
 - PACKING LIST : address not readable on printing
 - HUMAN RESOURCE DEMO DATA : holiday planning is now correctly imported.
+- STOCKMOVE : catch NPE when print with icon a deleted move.
```

Comment:
=> catch NPE when printing a deleted move.

Reply:
the error was not triggered by the toolbar